webpack.config.js

```diff
@@ -22,25 +22,19 @@ entries.unshift('core-js/fn/symbol');
 entries.unshift('whatwg-fetch');
 entries.unshift('url-search-params-polyfill');
 
-const basePath = 'build/admin';
-
 module.exports = (env, argv) => ({ // eslint-disable-line no-undef
     entry: entries,
     output: {
-        path: path.resolve('public'),
-        filename: basePath + '/[name].[chunkhash].js',
+        path: path.resolve('public/build/admin'),
```

Comment:
This would indeed be easier, but the reason it wasn't done this way, is that the `ManifestPlugin` now outputs the wrong paths (the `/build/admin` part is missing), and the files are not found if you reload in the browser.

In addition to that a lot of entries are not part of the `manifest.json` anymore, which is also not a good sign...

Reply:
Or maybe even pass it using one of the other two variables? `env` probably doesn't work, since I believe it is just `development` or `production`, but `argv` sounds like it is about arguments you can pass inside :slightly_smiling_face: